treeherder/etl/runnable_jobs.py

```diff
@@ -117,49 +118,49 @@ def run(self):
         self.update_runnable_jobs_table(jobs_per_branch)
 
 
-def _taskcluster_runnable_jobs(decision_task_id):
+def _taskcluster_runnable_jobs(project, decision_task_id):
+    ret = []
+    tc_graph = {}
     if not decision_task_id:
+        decision_task_id = _query_latest_gecko_decision_task_id(project)
```

Comment:
Please can you add a test for this new functionality using `responses` to mock the response from the taskcluster index and/or graph URL.